deeplabcut/create_project/new.py

```diff
@@ -193,7 +193,7 @@ def create_new_project(
                     import subprocess
 
                     subprocess.check_call("mklink %s %s" % (dst, src), shell=True)
-                except OSError:
+                except OSError, subprocess.CalledProcessError:
```

Comment:
Just need this minor change:
```suggestion
                except (OSError, subprocess.CalledProcessError):
```